packages/frontend/core/src/modules/workbench/entities/view.ts

```diff
@@ -2,13 +2,25 @@ import { Entity, LiveData } from '@toeverything/infra';
 import type { Location, To } from 'history';
 import { Observable } from 'rxjs';
 
-import { createIsland } from '../../../utils/island';
 import { createNavigableHistory } from '../../../utils/navigable-history';
 import type { ViewScope } from '../scopes/view';
 
 export class View extends Entity {
   id = this.scope.props.id;
 
+  sidebarTabs$ = new LiveData<string[]>([]);
```

Comment:
🤔 Why not keep `Sidebar` entity but manage in view directly?

Reply:
I deleted the sidebar, and the sidebar is now managed by the view

Reply:
Yes, I meant that is it more intuitive to maintain a `Sidebar` entity for the logic of the sidebar, as  `view.sidebar`?